app/views/news_tematica/news_tematicas/_bloque_mas_leidos.html.haml

```diff
@@ -9,5 +9,4 @@
         %h5.titulo= link_to_con_estadisticas masleido.titulo, masleido.contenido_dominio_link, @news_tematica
         %p.resumen
           = raw masleido.descripcion
-          = link_to_con_estadisticas('Leer&nbsp;más&nbsp;>>', masleido.contenido_dominio_link, @news_tematica,
-            class: 'leer_mas')
+          = link_to_con_estadisticas('Leer&nbsp;más&nbsp;>>'.html_safe, masleido.contenido_dominio_link, @news_tematica, class: 'leer_mas')
```

Comment:
I created the bug 2 weeks ago on (https://github.com/Soluciones/news_tematica/commit/b8b969038877c9e6259ee40578fd64238d6ce62e) when I missed `.html_safe`